### PR TITLE
Add paragliding icon

### PR DESCRIPTION
**Pull request by seg-on**
Added paragliding icon for sport=free_flying

### CODE REVIEW

File: images/free_flying.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14">
+ <g transform="translate(-187.1 -49.85)">
```
Comment (zdila) [inline images/free_flying.svg]: please get rid of transformations (eg. use ungroup function in inkscape and then minify SVG again)

I am also missing a white glow as we use for other icons.

File: legend.js [hunk 2]
```diff
@@ -838,6 +838,7 @@ const legend = {
     poi('poi', 'mine', 'mine, adit, mineshaft', 'baňa, štôlňa, šachta'),
     poi('poi', 'attraction', 'attraction', 'atrakcia'),
     poi('sauna', 'sauna', 'sauna'),
+	poi('free_flying', 'paragliding', 'paraglajding'),
```
Comment (zdila) [inline legend.js]: pls use spaces, not tabs

**Review by zdila — CHANGES_REQUESTED** [review]

**Comment by seg-on** [comment]
see if it's okay

File: style/index.js [hunk 3]
```diff
@@ -207,7 +207,11 @@ function generateFreemapStyle({
       .area(colors.allotments, 'allotments')
       .area(hsl(0, 0, 80), 'industrial', 'wastewater_plant')
       .area(hsl(320, 40, 85), 'commercial', 'retail')
-      .area(colors.wetland, 'wetland')
+      .area(colors.grassy, 'wetland')
```
Comment (zdila) [inline style/index.js:210]: toto mazem lebo to renderuje wetland vsade. prosim pred pull-requestom otestuj ci to funguje

**Comment by zdila** [comment]
vdaka, len nabuduce prosim dokoncit a lokalne otestovat. support som donokncil v https://github.com/FreemapSlovakia/freemap-mapnik/commit/da594304a12dc0089108a5d50c6179b914226867

**Comment by seg-on** [comment]
bohužiaľ nemám teraz na čom by som to rozbehal a testoval

**Pull request merged**